rust/kcl-lib/tests/angled_line/artifact_commands.snap

```diff
@@ -130,6 +130,17 @@ description: Artifact commands angled_line.kcl
       }
     }
   },
+  {
+    "cmdId": "[uuid]",
+    "range": [
+      35,
+      67,
+      0
+    ],
+    "command": {
+      "type": "sketch_mode_disable"
+    }
```

Comment:
WTF. This can't be right here, can it? Disable sketch mode before more `extend_path` commands.

Reply:
ben fixed it so we can :)

Reply:
this is needed for parallel modules otherwise we would be in a weird sketch mode for something else! it was a 8 line change in engine to make ti work :)

Reply:
Oh, interesting!

Reply:
in the limit we remove the whole "entering a mode" but this cleans it up greatly and now we atomically enter when we have to and every where else unnecessary